back/api/models.py

```diff
@@ -13,6 +13,9 @@ class Meta:
         db_table = 'copyright'
         verbose_name = _('copyright')
 
+    def __str__(self):
+        return self.description
```

Comment:
I do not get why this is needed... but... If you put it everywhere, then there is surely some kind of purpose...

Reply:
This overrides the __str__ method meaning when you print it, you'll not get `<__main__.Order instance at 0x7ff2a18c>` but instead you can get what you put in that return. Django relies on `__str__`  for showing lists, instances, etc...

![admin](https://user-images.githubusercontent.com/3328875/79952605-f89aea00-847a-11ea-801c-4049a92afd73.png)